content/spin/v2/cli-reference.md

```diff
@@ -707,6 +860,26 @@ OPTIONS:
 > Please note: Spin `help` is a convenient way to access help using a subcommand, instead of using the `--help` option. For example, `spin help cloud` will give you the same output as `spin cloud --help`. Similarly, `spin help build` will give you the same output as `spin build --help` and so forth.
 
 {{ blockEnd }}
+
+{{ startTab "v2.5"}}
+
+> Please note: Spin `help` is a convenient way to access help using a subcommand, instead of using the `--help` option. For example, `spin help cloud` will give you the same output as `spin cloud --help`. Similarly, `spin help build` will give you the same output as `spin build --help` and so forth.
+
+{{ blockEnd }}
+
+{{ blockEnd }}
+
+<!-- markdownlint-disable-next-line titlecase-rule -->
+## spin kube
+
+{ tabs "spin-version" }}
+
+{{ startTab "v2.5"}}
```

Comment:
I don't think `spin kube` is tied to 2.5?

Reply:
I could back-date this for 2.4 also, but it is probably not relevant anymore since we are past that point.